addons/website_forum/views/website_forum.xml

```diff
@@ -142,129 +127,205 @@
-                    <span t-if="not question.active and question.state=='offensive'"><b> [Offensive]</b></span>
-                    <span t-if="not question.active and question.state=='offensive' and question.closed_reason_id"><b> [<t t-esc="question.closed_reason_id.name[0].upper() + question.closed_reason_id.name[1:]"/>]</b></span>
-                    <span t-if="question.state == 'close'"><b> [Closed]</b></span>
-                </div>
-                <div><t t-raw="post_content"/></div>
-                <small class="text-muted">
-                    <i t-attf-class="fa fa-question-circle" role="img" aria-label="Question" />
-                    <span>By </span><span t-field="question.create_uid" t-options='{"widget": "contact", "country_image": True, "fields": ["name", "country_id"]}' class="d-inline-block"/>
-                    <span class="ml-4 mr-1 fa fa-calendar"></span><span t-field="question.write_date" t-options='{"format":"short"}'/>
-                    <span class="ml-4 mr-1 fa fa-eye"></span><span t-field="question.views" /> <t t-if="question.views&lt;=1">View</t><t t-else="">Views</t>
-                    <i class="ml-4 fa fa-comments text-primary"/>
-                    <a t-attf-href="/forum/#{ slug(question.forum_id) }/question/#{ slug(question) }">
-                        <t t-esc="question.child_count"/>
-                        <t t-if="question.child_count&lt;=1">Answer</t>
-                        <t t-else="">Answers</t>
-                    </a>
-                    <a t-attf-href="/forum/#{ slug(question.forum_id) }/question/#{ slug(question) }">
-                        <span class="ml-4 mr-1 fa fa-star"></span><span t-field="question.favourite_count"/> <t t-if="question.views&lt;=1">Favorite</t><t t-else="">Favorites</t>
-                    </a>
-                    <span t-if="question.state == 'flagged'" class="text-black">
-                        <i class="fa fa-flag ml-4 mr4"/>
-                        Flagged
-                    </span>
-                </small>
-            </div>
+
+    <div t-attf-class="o_wforum_index_entry_tags mb-1" t-if="len(question.tag_ids) > 0">
+        <t t-foreach="question.tag_ids" t-as="question_tag">
+
+            <!-- Toggle Tags on click -->
+            <t t-if="tag and tag.name == question_tag.name" t-set="click_action"
+                t-value="'/forum/' + slug(question_tag.forum_id) + '?' + keep_query( 'search', 'sorting', 'my')"/>
+            <t t-else="" t-set="click_action"
+                t-value="'/forum/' + slug(question_tag.forum_id) + '/tag/' + slug(question_tag) + '/questions?' + keep_query( 'search', 'sorting', 'my', filters='tag')"/>
+
+            <a t-att-href="click_action"
+                t-attf-class="badge #{tag and tag.name == question_tag.name and 'badge-secondary' or 'border text-600 badge-light'} #{'ml-lg-1 mt-lg-1' if not (question_tag_first) else ''}"
```

Comment:
#{'ml-lg-1 mt-lg-1' if not (question_tag_first) else ''}
 #{not question_tag_first and 'ml-lg-1 mt-lg-1'}